ui/src/app/views/workflowv3/run/workflowv3-run-job.component.ts

```diff
@@ -0,0 +1,114 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, Input, OnChanges, Output, ViewChild } from '@angular/core';
+import { Parameter } from 'app/model/parameter.model';
+import { WorkflowNodeJobRun } from 'app/model/workflow.run.model';
+import { WorkflowRunService } from 'app/service/services.module';
+import { AutoUnsubscribe } from 'app/shared/decorator/autoUnsubscribe';
+import { Tab } from 'app/shared/tabs/tabs.component';
+import { ScrollTarget } from 'app/views/workflow/run/node/pipeline/workflow-run-job/workflow-run-job.component';
+import { finalize } from 'rxjs/operators';
+import { JobRun } from '../workflowv3.model';
+
+@Component({
+    selector: 'app-workflowv3-run-job',
+    templateUrl: './workflowv3-run-job.html',
+    styleUrls: ['./workflowv3-run-job.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+@AutoUnsubscribe()
+export class WorkflowV3RunJobComponent implements OnChanges {
+    @ViewChild('workflowRunJob', { read: ElementRef }) workflowRunJob: ElementRef;
+
+    @Input() projectKey: string;
+    @Input() workflowName: string;
+    @Input() workflowRunNum: number;
+    @Input() jobRun: JobRun;
+
+    @Output() onClickClose = new EventEmitter<void>();
+
+    tabs: Array<Tab>;
+    selectedTab: Tab;
+    loading = false;
+    selectedNodeJobRun: WorkflowNodeJobRun;
+    variables: { [key: string]: Array<Parameter> } = {};
+    variableKeys: Array<string> = [];
+
+    constructor(
+        private _cd: ChangeDetectorRef,
+        private _workflowRunService: WorkflowRunService
+    ) {
+        this.tabs = [<Tab>{
+            translate: 'common_logs',
+            key: 'logs',
+            default: true
+        }, <Tab>{
+            translate: 'common_variables',
+            key: 'variables'
+        }];
+    }
+
+    ngOnChanges(): void {
```

Comment:
Can't catch event on node run change? ( already in the websocket )

Reply:
Done